index.js

```diff
@@ -18,9 +18,6 @@ const commands = (url) => {
 const open = url => new Promise((resolve, reject) => {
   exec(commands(url), (error, stdout, stderr) => {
     if (error) return reject(error);
-    /* eslint-disable no-console */
-    console.log(`stdout: ${stdout}`);
-    console.log(`stderr: ${stderr}`);
```

Comment:
Instead of removing them totally, it will be good to have them as optional

Reply:
Hey @tacheometry
Thanks for your contribution.
This has been auto fixed in [this](https://github.com/azhar22k/ourl/pull/9) PR where `exec` was replaced by `execSync`